tests/test_field.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 from django.core.exceptions import ValidationError
-from pytest_lazyfixture import lazy_fixture
+from pytest_lazy_fixtures import lf
```

Comment:
Given that `lf` is a bit cryptic (is it linefeed?) what about...
```suggestion
from pytest_lazy_fixtures import lf as lazy_fixture
```
and then revert all the remaining changes in this file.